addons/im_livechat/static/src/embed/frontend/livechat_root.js

```diff
@@ -19,5 +19,7 @@ export class LivechatRoot extends Component {
     setup() {
         useSubEnv({ embedLivechat: true });
         this.overlayService = useService("overlay");
+        this.store = useService("mail.store");
+        this.store.embedLivechat = true;
```

Comment:
`embedLivechat` in `env` is set by livechat root at component level, therefore it's not available in service `env`. That's why I defined `store.embedLivechat` for model-level check of `embedLivechat`.